.evergreen.yml

```diff
@@ -1,3 +1,4 @@
+# Regenerate using `npm run update-evergreen-config`
```

Comment:
This file is automatically generated now – most of the diff here is cosmetic changes.